pages/common/gource.md

```diff
@@ -0,0 +1,31 @@
+# gource
+
+> Visualize the history of Git, SVN, Mercurial and Bazaar repositories.
```

Comment:
Perhaps mention that it produces a tree diagram of the project's directory structure, animated to show files and folders being created, modified or removed over time. Or some other description, hopefully more concise :)